lib/d3/readme_moodle.txt

```diff
@@ -0,0 +1,7 @@
+
+Procedure for including D3 library into Moodle.
+
+1. The download website is https://d3js.org.
+2. Download the latest version in a tgz file via a link given on the above page.
+3. Extract from the tgz file and move the contents of the 'package/dist' directory and LICENSE into lib/d3 of the plugin.
+
```

Comment:
Similarly to the previous point regarding d3-flame-graph, could you make this into a list of commands and the steps as command comments?

An example URL would be nice to go through the update process as easily as possible without needing to do much manual work.